packages/mdc-fab/package-lock.json

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
we don't have package-lock.json files in packages.

Reply:
Removed.